cmd/ingress/ingress.go

```diff
@@ -166,6 +166,7 @@ For example, no available LB exists in the bare metal environment.`)
 	cmd.PersistentFlags().StringVar(&cfg.Kubernetes.ApisixTlsVersion, "apisix-tls-version", config.ApisixV2beta3, "the supported apisixtls api group version, can be \"apisix.apache.org/v2beta3\" or \"apisix.apache.org/v2\"")
 	cmd.PersistentFlags().StringVar(&cfg.Kubernetes.ApisixClusterConfigVersion, "apisix-cluster-config-version", config.ApisixV2beta3, "the supported ApisixClusterConfig api group version, can be \"apisix.apache.org/v2beta3\" or \"apisix.apache.org/v2\"")
 	cmd.PersistentFlags().StringVar(&cfg.Kubernetes.ApisixConsumerVersion, "apisix-consumer-version", config.ApisixV2beta3, "the supported ApisixConsumer api group version, can be \"apisix.apache.org/v2beta3\" or \"apisix.apache.org/v2\"")
+	cmd.PersistentFlags().StringVar(&cfg.Kubernetes.ApiVersion, "api-version", config.DefaultApisixVersion, config.ApisixVersionDescribe)
```

Comment:
Can you replace these `ApisixVersion` with `APIVersion`?  

We don’t need keep the name `config.ApisixVersion` `config.DefaultApisixVersion` etc.

Reply:
Sure!